celestia/peers.txt

```diff
@@ -1 +1 @@
-
+5b0d1b8cb6f893426168300f69a82750d4be8729@193.35.57.185:11656
```

Comment:
please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`